README.md

```diff
@@ -16,3 +57,12 @@ $ npm install
 $ npm link @balena/gsheet-utils
```

Comment:
Change to
```
npm link @balena/dreamsheets
```
or
```
npm link dreamsheets
```
depending on the value of `name` in `package.json`.

Reply:
@AlidaOdendaal I've updated to `npm link dreamsheets`

Reply:
@AlidaOdendaal's review is resolved, so I'll save her a trip here and self-certify.